wallet/internal/db/migrations/postgres/000002_wallets.up.sql

```diff
@@ -0,0 +1,69 @@
+-- Wallet metadata and related state tables.
+--
+-- Migration note: Intentionally NOT idempotent (no "IF NOT EXISTS").
+-- This ensures migration tracking stays accurate and fails loudly if run twice.
+CREATE TABLE wallets (
+    -- DB ID of the wallet, primary key. Only used for DB level relations.
+    id BIGSERIAL PRIMARY KEY,
+    -- Human friendly name for the wallet.
+    name TEXT NOT NULL,
+    -- Defines if the wallet was imported, so all accounts should be imported.
```

Comment:
You mean when the wallet is imported, all its accounts are also considered imported right?


```suggestion
    -- Defines if the wallet was imported, all its accounts would also be imported.
```

Reply:
yes, changed